source/agg-pipeline-builder/create-a-view.txt

```diff
@@ -12,5 +12,44 @@ Create a View from Pipeline Results
    :depth: 1
    :class: singlecol
 
-TODO: 
+To quickly access the results of an aggregation pipeline without having to 
+execute an aggregation pipeline, you can create a :ref:`view <compass-views>` 
+on |compass|. Views are read-only, so they can help keep your data secure by 
+limiting user access to a predefined set of results. 
+
+About This Task 
+---------------
+
+Creating a view does not save the aggregation pipeline itself. 
+
+Procedure
+---------
+
+.. procedure:: 
+   :style: normal
+
+   .. step:: Click the Save dropdown button
+
+      In the aggregation pipeline pane, click the :guilabel:`Save` drop-down 
+      menu and select :guilabel:`Create view`.
+
+      .. figure:: /images/compass/query-save-pipeline-as.png
+         :alt: Save drop-down menu
+
+   .. step:: Enter a name for your view
+
```

Comment:
(suggestion)
If you include "Enter a name for your view" as a step, let's add a step description. You could mention if there are special character or length restrictions for the view name? What other view naming guidelines might be useful?

Alternately, you could make "Create your view" step 2, and add two substeps. Something like:
2. Create your view
      a. Enter a name for your view.
      b. Click **Create**.

Reply:
Updated!